.appveyor.yml

```diff
@@ -1,49 +1,62 @@
-image: Visual Studio 2017
+image:
+  - Ubuntu
+  - Visual Studio 2017
 version: 2.2.0-{build}
-services:
-  - postgresql101
-environment:
-  global:
-    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
-    DOTNET_CLI_TELEMETRY_OPTOUT: 1
-    Test__Npgsql__DefaultConnection: Host=localhost;Database=postgres;Username=postgres;Password=Password12!
-    PGUSER: postgres
-    PGPASSWORD: Password12!
-    POSTGIS_EXE: postgis-bundle-pg10x64-setup-2.4.4-1.exe
-    NoPackageAnalysis: true  # Suppresses warning about SemVer 2.0.0 version suffixes when packing
-cache:
-  - '%USERPROFILE%\.nuget\packages -> **\*.csproj'
-  - $(POSTGIS_EXE)
 clone_depth: 10
+environment:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  PGPASSWORD: Password12! # Used by psql on Windows.
+  POSTGIS_EXE: postgis-bundle-pg10x64-setup-2.4.4-1.exe
+  NoPackageAnalysis: true  # Suppresses warning about SemVer 2.0.0 version suffixes when packing
+for:
+-
+  matrix:
+    only:
+      - image: Visual Studio 2017
+  cache:
+    - '%USERPROFILE%\.nuget\packages -> **\*.csproj'
+    - $(POSTGIS_EXE)
+  services:
+    - postgresql101
+-
+  matrix:
+    only:
+      - image: Ubuntu
+  cache:
+    - ~/.nuget/packages -> **/*.csproj
+  services:
+    - postgresql
 install:
-  - powershell .build\setup_appveyor.ps1
-  # The following can be used to install a custom version of .NET Core
-  # - ps: Invoke-WebRequest -Uri "https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1" -OutFile "install-dotnet.ps1"
-  # - ps: .\install-dotnet.ps1 -Version 2.1.300-rc1-008673
-  #
-  # - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+  - pwsh: .build/install.ps1
```

Comment:
My thinking is that since `pwsh` uses PS Core in both builds, this let's us keep common setup in the `pwsh` rather than duplicating in separate scripts.

Reply:
[According to the docs](https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/#bash-and-powershell) it seems that `pwsh` and `ps` are synonyms? Or am I missing something?

Reply:
Sigh, the trouble with their documentation is that its incomplete for the Linux side. 

My reading of that is that for _Linux_ they're interchangeable, but I couldn't find a citation for them being interchangeable on Windows (where both Windows PowerShell and PowerShell Core are installed).

I'll swap them out and see if it still works. It would be very nice if my interpretation was overly pessimistic. It's visually clearer with consistent-width prefixes (e.g. `ps` and `sh`).

Reply:
`pwsh` is the official name of PowerShell Core, while `ps` is a Linux command. Therefore, it would be better to use `pwsh` to avoid ambiguity.

Reply:
Actually, now that `sh` is only called in an `if`/`else` branch, the alignment doesn't matter:

```yaml
install:
  - ps: .build/set_version.ps1
  - ps: |
      if ($isLinux) {
        sh .build/install.sh
      } else {
        .build/install.ps1
      }
```
```yaml
install:
  - pwsh: .build/set_version.ps1
  - pwsh: |
      if ($isLinux) {
        sh .build/install.sh
      } else {
        .build/install.ps1
      }
```

+1 for the verbosity/obviousness of `pwsh`, but I'm good either way.